File: py_coding_q_and_a/09.py
# find the occurances count of all character in a text

text = "Hello javascript python"

def count_chars(text):
    res={}
    text.replace(" ","")
    for char in text:
        if char == " ": continue
        if char in res:
            res[char]+=1
        else:
            res[char]=1
    return res

print(count_chars(text))

File: js_coding_q_and_a/09.js
// find the occurances count of all character in a text

var text = "Hello javascript";

text = text.replace(" ", "").toLowerCase().split("");

var res = text.reduce((acc, cur) => {
  acc[cur] = (acc[cur] | 0) + 1;
  return acc;
}, {});
console.log(res);
